@jozefizso Thanks for your PR. Now `ignition-msgs5` build failed with the following error, it build depends on `protobuf`.
`buildtrees\ignition-msgs5\src\sgs5_5.1.0-e593042925\src\Utility.cc : fatal error C1128: number of sections exceeded object file format limit: compile with /bigobj`

--- a/ports/ignition-msgs5/01-protobuf.patch
+++ b/ports/ignition-msgs5/01-protobuf.patch
@@ -1,0 +1,13 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index f68bf3d..d4d1372 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -135,6 +135,8 @@ if(MSVC)
+   # warning is not important since those members do not need to be interfaced
+   # with.
+   set_source_files_properties(${gen_sources} COMPILE_FLAGS "/wd4251 /wd4146")
++  # Fix for protobuf 3.12 - allow big object files
++  add_definitions(/bigobj)
+ endif()
+ 
+ set_source_files_properties(${gen_headers} ${gen_sources} ${gen_ruby_scripts}

--- a/ports/ignition-msgs5/portfile.cmake
+++ b/ports/ignition-msgs5/portfile.cmake
@@ -12,7 +12,9 @@ vcpkg_add_to_path(${CURRENT_INSTALLED_DIR}/debug/bin)
 
 ignition_modular_library(NAME msgs
                          VERSION "5.1.0"
-                         SHA512 db485f737c465b310602342a1a751c561473e330eb18d1b322b32d13de246536fe6a7efdf245faaaa9be1c9bfce662b2d39d1bb7cffc37e52690893cb47cc2ee)
+                         SHA512 db485f737c465b310602342a1a751c561473e330eb18d1b322b32d13de246536fe6a7efdf245faaaa9be1c9bfce662b2d39d1bb7cffc37e52690893cb47cc2ee
+                         PATCHES
+                           "01-protobuf.patch")
 
 # Restore old path
 set(ENV{PATH} ${path_backup})

